src/web/layouts/DecideLayout.tsx

```diff
@@ -73,6 +73,15 @@ export const DecideLayout = ({ CAPI, NAV }: Props) => {
         case Display.Showcase: {
             switch (designType) {
                 case 'Comment':
+                    return (
+                        <CommentLayout
```

Comment:
I don't think this is comment only, I believe any article layout can have showcase images. I.E: https://www.theguardian.com/science/grrlscientist/2012/aug/07/3

Reply:
Not sure whether this was opinion specific in this instance?